app/src/features/pricing/constants/pricing.js

```diff
@@ -30,6 +31,8 @@ PRICING.PLAN_NAMES.PROFESSIONAL_ENTERPRISE = "professional_enterprise";
 PRICING.PLAN_NAMES.TRIAL_EXPIRED = "trial_expired";
 PRICING.PLAN_NAMES.FREE = "free";
 PRICING.PLAN_NAMES.INDIVIDUAL = "individual";
+PRICING.PLAN_NAMES.API_CLIENT_PROFESSIONAL = "api_client_professional";
+PRICING.PLAN_NAMES.API_CLIENT_ENTERPRISE = "api_client_enterprise";
```

Comment:
@Ethical98 Lets keep them as api-client-professional and api-client-enterprise. planIds shouldn't use`_` as we split by `_` to get the plan Names from planIds for legacy plans